src/Utilities/Testing.hpp

```diff
@@ -15,19 +15,6 @@
 
 #include <PowerSystemData.hpp>
```

Comment:
`PowerSystemData.hpp` no longer needed here.

Reply:
Good catch, thanks!

Reply:
Done in [35daa10](https://github.com/ORNL/GridKit/pull/110/commits/35daa10443545bb0529132e42692a4e92f926cdd).